resource/ui/ts/models/node_status.ts

```diff
@@ -141,10 +139,70 @@ module Models {
                         m("tr", [m("td", "Ranges:"), m("td", node.range_count)]),
                         m("tr", [m("td", "Leader Ranges:"), m("td", node.leader_range_count)]),
                         m("tr", [m("td", "Available Ranges:"), m("td", node.available_range_count)]),
-                        m("tr", [m("td", "Availablility:"), m("td", this._availability(nodeId))]),
+                        m("tr", [m("td", "Availablility:"), m("td", Nodes._availability(node))]),
                         m("tr", [m("td", "Under-Replicated Ranges:"), m("td", node.leader_range_count - node.replicated_range_count)]),
-                        m("tr", [m("td", "Fully Replicated:"), m("td", this._replicated(nodeId))])
-                    ])
+                        m("tr", [m("td", "Fully Replicated:"), m("td", Nodes._replicated(node))])
+                    ]),
+                    Stats.CreateStatsTable(node.stats)
+                ]);
+            }
+
+            public AllDetails(): _mithril.MithrilVirtualElement {
+
+                var status = <NodeStatus>{
+                    range_count: 0,
+                    updated_at: 0,
+                    leader_range_count: 0,
+                    replicated_range_count: 0,
+                    available_range_count: 0,
+                    stats: <Models.Stats.MVCCStats>{
+                        live_bytes: 0,
+                        key_bytes: 0,
+                        val_bytes: 0,
+                        intent_bytes: 0,
+                        live_count: 0,
+                        key_count: 0,
+                        val_count: 0,
+                        intent_count: 0,
+                        sys_bytes: 0,
+                        sys_count: 0
+                    }
+                };
+
+                var nodeId: string;
+                for (nodeId in this.statuses()) {
+                    var nodeStatus = this.statuses()[nodeId];
+                    status.range_count += nodeStatus.range_count;
+                    status.leader_range_count += nodeStatus.leader_range_count;
+                    status.replicated_range_count += nodeStatus.replicated_range_count;
+                    status.available_range_count += nodeStatus.available_range_count;
+                    if (nodeStatus.updated_at > status.updated_at) {
+                        status.updated_at = nodeStatus.updated_at;
+                    }
+                    status.stats.live_bytes += nodeStatus.stats.live_bytes;
```

Comment:
this field-by-field addition seems fraught with danger. is it possible to implement the addition operator in TS? might be good to do in a future PR.